src/resources/auth/auth.validator.ts

```diff
@@ -0,0 +1,35 @@
+import {body} from 'express-validator/check'
+import {returnValidationError} from '../../middlewares/error'
+
+/**
+ * Validate sign up request
+ */
+export const validateSignUp = () => {
+	return [
+		body('email', 'Invalid email')
+			.isEmail()
+			.normalizeEmail(),
+		body('password', 'Password must be at least 5 chars long').isLength({
+			min: 5,
+		}),
+		body('firstName', 'First name should not be empty').exists(),
```

Comment:
I think exists() need checkFalsy option as well, pure exist() only check for undefined value
https://express-validator.github.io/docs/validation-chain-api.html#existsoptions